packages/launcher/lib/windows/index.ts

```diff
@@ -185,6 +185,29 @@ export function getVersionString (path: string) {
   .then(trim)
 }
 
+export function getPathData (pathStr: string): PathData {
```

Comment:
[As mentioned](https://github.com/cypress-io/cypress/issues/6389#issuecomment-632706124), this should automatically infer `chrome.exe` to be `chrome` and `firefox.exe` to be `firefox`, since those are the default names for the executables. This will allow most users to avoid having to use or learn about the `:browserName` solution.